lib/shifts.rb

```diff
@@ -6,12 +6,12 @@ def random_number
       rand(99999)
     end
 
-    def padded_random_number(number = random_number)
+    def padded_number(number = random_number)
```

Comment:
Renamed method so it made more sense with given numbers which are inherently NOT random.